src/components/panes/EditCallerPane.jsx

```diff
@@ -74,18 +74,18 @@ export default class EditCallerPane extends PaneBase {
                 onAdd={ this.onAddTag.bind(this, 'ex') }
                 onRemove={ this.onRemoveExTag.bind(this) }/>
         }
-
+        
         return [
             <h3 key="prioHeader">
-                <i className="fa fa-star"/>
-                <Msg id="panes.editCaller.prio.h"/>,
+                <i className="fa fa-star" style = {{padding : "10px"}}/>
```

Comment:
Nice solution, but we tend to prefer CSS. Try this instead:

Create a file next to `EditCallerPane.jsx` called `EditCallerPane.scss` and add the following SASS:

```scss
.EditCallerPane {
    h3 {
        i {
            margin: 0 1em 0 0;
        }
    }
}
```

This also only adds padding to the right of the icon, which I think looks nicer than adding it on all sides.